lib/wordpress.js

```diff
@@ -0,0 +1,48 @@
+var _ = require('underscore');
+var params = require('querystring').parse(window.location.search.split('?').pop());
+
+module.exports = {
+  getPostPreviewView: function (view) {
+    // Return false if we don't have everything we need for a preview
+    var isPreviewActive = params._wp_json_nonce && params.is_preview;
+    if (!isPreviewActive) {
+      console.info('Preview is not active.');
```

Comment:
This will throw errors in IE 9 (and below), and it's not standard JS still so I think it's best to leave it out.